configuration/index.md

```diff
@@ -128,14 +129,39 @@ Every interface has it's own strong points.
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.X, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI". It allows for configuration of the base system and setup of openHAB 2
+addons. Please be aware that not every addon/binding that has been existing since openHAB version 1 has been updated to version 2 and
+thus to work with PaperUI. You can still run these but need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a 
+`.items` configuration file. Configuration done in files will be synced to the database (so it's accessible in PaperUI) but
+configuration done in the database will not be written back to files, so if you restart openHAB, changes made in PaperUI won't survive 
+and the files' configuration data will be used again.
+
+## A recommendation
+
+All interface options are supposed to work, it's just that some of the more advanced settings might work better in one or the other
```

Comment:
Please clean up with whole section wrt the spelling of PaperUI and line breaks (which must be behind a sentence).
Also avoid talking of "we", rather use passive voice instead. Do not abbreviate with "'", use full words (you're -> you are, etc.).